ebpf-profiling.graphqls

```diff
@@ -30,6 +30,15 @@ input EBPFProfilingTaskFixedTimeCreationRequest {
     targetType: EBPFProfilingTargetType!
 }
 
+# The request of eBPF network topology profiling ondemand task
+input EBPFProfilingOnDemandNetworkTopologyTaskRequest {
```

Comment:
Shouldn't this be named `networkProfiling` or something?

Reply:
Please remove `onDemand` out of name. All profiling tasks are on-demand.

Reply:
Update to `EBPFProfilingNetworkTaskRequest`.